### ISSUE TITLE
`az storage blob upload-batch`: ERROR: HTTPSConnectionPool

### ISSUE THREAD

**Issue by bacongobbler** [Azure/azure-cli#4223]
### Description

When running `az storage blob upload-batch --source _dist/ --destination draft --pattern *.tar.gz*`, the following error consistently pops up with no descriptive error:

```
+ az storage blob upload-batch --source _dist/ --destination draft --pattern *.tar.gz*
ERROR: 
HTTPSConnectionPool
uploading /home/jenkins/go/src/github.com/Azure/draft/_dist/draft-git-92bbd83-windows-amd64.tar.gz
```

It never actually uploads anything up to Azure blob storage. It just appears to error out.

This is running in our CI system, which is able to connect to the outside world and upload Docker images to hub.docker.com, so at first glance it doesn't seem like a connection issue. Is there anything we can do on our end to figure out the root cause of this issue from within the CLI?

### Environment summary

**Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, nightly)  
pip

**CLI Version:** What version of the CLI and modules are installed? (Use `az --version`)  

```
><> docker run --entrypoint="/bin/az" deis/jenkins-node:v0.4.4 -v
azure-cli (2.0.13)

acr (2.0.10)
acs (2.0.12)
appservice (0.1.12)
batch (3.1.1)
billing (0.1.3)
cdn (0.0.6)
cloud (2.0.7)
cognitiveservices (0.1.6)
command-modules-nspkg (2.0.1)
component (2.0.7)
configure (2.0.10)
consumption (0.1.3)
container (0.1.8)
core (2.0.13)
cosmosdb (0.1.11)
dla (0.0.10)
dls (0.0.12)
eventgrid (0.1.1)
feedback (2.0.6)
find (0.2.6)
interactive (0.3.7)
iot (0.1.10)
keyvault (2.0.8)
lab (0.0.9)
monitor (0.0.8)
network (2.0.12)
nspkg (3.0.1)
profile (2.0.10)
rdbms (0.0.5)
redis (0.2.7)
resource (2.0.12)
role (2.0.10)
sf (1.0.6)
sql (2.0.9)
storage (2.0.12)
vm (2.0.12)

Python (Linux) 3.6.1 (default, Aug 11 2017, 17:14:09) 
[GCC 4.8.4]

Python location '/opt/az/bin/python3'
```

**OS Version:** What OS and version are you using?

```
><> docker run --entrypoint="/bin/uname" deis/jenkins-node:v0.4.4 -a
Linux a83628dcb6d7 4.9.36-moby #1 SMP Wed Jul 12 15:29:07 UTC 2017 x86_64 GNU/Linux
```

**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, Bash on Windows)  
bash

**Comment by bacongobbler** [comment]
Attaching a screenshot to confirm no archives were uploaded to Azure. 
<img width="2560" alt="screen shot 2017-08-15 at 1 57 37 pm" src="https://user-images.githubusercontent.com/1360539/29336184-dd9344be-81c1-11e7-8520-f81cc5b30621.png">

**Comment by bacongobbler** [comment]
fixed by upgrading to v2.0.15.

**Comment by lmiroslaw** [comment]
I see the same error when running _az storage blob upload_ .
